frontend/src/Components/HomePage/UpcomingTrips/AltFlightModal.jsx

```diff
@@ -1,45 +1,120 @@
+import React, { useState, useMemo } from 'react';
 import './AltFlightModal.css';
 import PropTypes from 'prop-types';
 
 function AltFlightModal({ isOpen, onClose, flights }) {
+    const [activeTab, setActiveTab] = useState('Best');
+
+    const formatTime = (dateString) => {
+        const date = new Date(dateString);
+        return date.toLocaleTimeString('en-US', { hour: 'numeric', minute: '2-digit', hour12: true }).toLowerCase();
+    };
+
+    const sortedFlights = useMemo(() => {
+        if (!flights) return [];
+        
+        switch (activeTab) {
+            case 'Best':
```

Comment:
It is recommended to use enums:

https://www.w3schools.com/typescript/typescript_enums.php

No need to change it now, but as best practice, we should usually use enums to avoid human mistakes and make refactors easier.